.azure-pipelines/pipelines.yml

```diff
@@ -635,6 +635,14 @@ stages:
         GCP_SERVICE_ACCOUNT_KEY: $(GcpServiceAccountKey)
       displayName: "Generate docs"
 
+    - script: |
+        ci/run_envoy_docker.sh 'bazel run //distribution/dockerhub:update_repository'
+      displayName: "Publish Dockerhub description and README"
+      env:
+        DOCKERHUB_USERNAME: $(DockerUsername)
+        DOCKERHUB_PASSWORD: $(DockerPassword)
```

Comment:
@lizan for this to work the `PAT` has to have `admin` permissions

Reply:
~checking the existing creds and it doesnt have this it will need to be updated before this can land~

~its unfortunate that the dockerhub API require so much permission to update the description~

according to this comment https://github.com/docker/hub-feedback/issues/2127#issuecomment-1183687657 the current permissions should be sufficient now

Reply:
testing this further and to use a PAT it must have the read/write/delete permissions